specs/phase1/custody-game.md

```diff
@@ -360,13 +413,16 @@ def process_custody_slashing(state: BeaconState, signed_custody_slashing: Signed
     assert is_valid_indexed_attestation(state, get_indexed_attestation(state, attestation))
 
     # TODO: custody_slashing.data is not chunked like shard blocks yet, result is lots of padding.
+    # ??? What does this mean?
```

Comment:
@protolambda I didn't really understand that comment, what was that about?

Reply:
It's outdated I think, but has to do with: https://github.com/ethereum/eth2.0-specs/commit/2a91b43eaf6d883750216d82e921227b03be3223
There was a chunking approach to split a block in 4 parts, and the slashing code did not match.